bin/improver-nbhood

```diff
@@ -73,22 +85,60 @@ def main():
                         'every grid square is considered to be the '
                         'equivalent of an ensemble member.'
                         'Optional, defaults to 1.0.')
+    parser.add_argument('--weighted_mode', action='store_true', default=False,
+                        help='For neighbourhood processing using a circular '
+                             'kernel, setting the weighted_mode indicates the '
+                             'weighting decreases with radius. '
+                             'If weighted_mode is not set, a constant '
+                             'weighting is assumed. weighted_mode is only '
+                             'applicable for calculating "probability" '
+                             'neighbourhood output.')
+    parser.add_argument('--percentiles', metavar='PERCENTILES',
+                        default=DEFAULT_PERCENTILES, nargs='+',
+                        help='Calculate values at the specified percentiles '
+                             'from the neighbourhood surrounding each grid '
+                             'point.')
     parser.add_argument('input_filepath', metavar='INPUT_FILE',
                         help='A path to an input NetCDF file to be processed.')
     parser.add_argument('output_filepath', metavar='OUTPUT_FILE',
                         help='The output path for the processed NetCDF.')
     args = parser.parse_args()
+
+    if (args.neighbourhood_output == "percentiles" and
```

Comment:
Do we need any tests for testing it fails when wrong arguments or not?

Reply:
I haven't added any CLI tests to test incorrect arguments. Is this needed?

Reply:
I think its probably fine.